corehq/apps/geospatial/migrations/0008_geoconfig_travel_mode.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.11 on 2024-05-28 05:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('geospatial', '0007_geoconfig_max_case_distance_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='geoconfig',
+            name='travel_mode',
+            field=models.CharField(choices=[('walking', 'Walking'), ('cycling', 'Cycling'), ('driving', 'Driving')], default='driving', max_length=50),
+        ),
+    ]
```

Comment:
can we merge the two migrations into one?

Reply:
I can do that